src/cirrus/management/deployment.py

```diff
@@ -306,3 +310,45 @@ def template_payload(
             silence_templating_errors,
             **(additional_vars or {}),
         )
+
+    def yield_payloads(
+        self,
+        collections_workflow: str,
+        limit: int | None,
+        query_args: dict[str, str | None],
+        rerun: bool,
+    ) -> Iterator[dict]:
+        os.environ["CIRRUS_STATE_DB"] = self.environment["CIRRUS_STATE_DB"]
+        statedb = StateDB()
```

Comment:
This is works, but `StateDB` can take the `table_name` directly, rather than round tripping through `os.environ`:
```suggestion
        statedb = StateDB(table_name=self.environment["CIRRUS_STATE_DB"])
```

Reply:
added